odoo/addons/base/models/ir_actions.py

```diff
@@ -229,16 +229,16 @@ def _get_action_dict(self):
         """
         self.ensure_one()
         readable_fields = self._get_readable_fields()
-        if (self.sudo().type == "ir.actions.act_window"):
-            result = self.sudo().read()[0]
-            embedded_actions = self.env["ir.embedded.actions"].browse(result["embedded_action_ids"]).read()
-            result.update({"embedded_action_ids": embedded_actions})
-            return result
-        return {
+        result = {
             field: value
             for field, value in self.sudo().read()[0].items()
             if field in readable_fields
         }
+        if (self.sudo().type == "ir.actions.act_window"):
+            embedded_actions = self.env["ir.embedded.actions"].browse(result["embedded_action_ids"]).read()
+            result.update({"embedded_action_ids": embedded_actions})
+
+        return result
```

Comment:
```suggestion
        if self.sudo().type == "ir.actions.act_window":
            embedded_actions = self.env["ir.embedded.actions"].browse(result["embedded_action_ids"]).read()
            result.update({"embedded_action_ids": embedded_actions})
        return result
```
useless parenthesis and useless whitespace